### DISCUSSION TITLE
only key-bindings.bash I want use only the CTRL+R aka history search

### DISCUSSION THREAD

**Discussion by og900aero** [junegunn/fzf#4059] [Q&A] [answered]
If I want only the new CTRL+R search in the history, whar can I do? Maybe use key-bindings.bash? And how?

**Comment by LangLangBart** [comment, marked as answer]
#### key-bindings.bash

<kbd>⌃ Control</kbd><kbd>T</kbd> and <kbd>⌥ Option</kbd><kbd>C</kbd> can be disabled by setting their environment variable counterpart to an empty string[^1][^2].

```sh
FZF_CTRL_T_COMMAND= FZF_ALT_C_COMMAND= eval "$(fzf --bash)"
```

#### completion.bash

The completion part is only triggered with the `FZF_COMPLETION_TRIGGER` (default: `**`). If you want to avoid that, set it to an unlikely string.

```sh
export FZF_COMPLETION_TRIGGER="s05Dx46VJrKXPVMRvfRe9dfzvDTQ"
```

[^1]: [Setting up shell integration - junegunn/fzf: :cherry_blossom: A command-line fuzzy finder](https://github.com/junegunn/fzf#setting-up-shell-integration)
[^2]: [Make it possible to disable `Ctrl+T` / `Alt+C` / completions by elibarzilay · Pull Request #3678 · junegunn/fzf](https://github.com/junegunn/fzf/pull/3678)

**Reply by og900aero** [reply]
Thx, working fine.
